src/site/site.xml

```diff
@@ -71,7 +71,7 @@
 
         <menu name="Apache Curator" inherit="top">
             <item name="About" href="index.html"/>
-            <item name="Download" href="https://cwiki.apache.org/confluence/display/CURATOR/Releases"/>
+            <item name="Download" href="releases.html"/>
```

Comment:
Shall we name this tag as "Releases"?

Reply:
Every website has a "Download" page 
:-)
In my opinion it is better to name it "Download"